charts/falco/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: falco
-version: 4.21.1
-appVersion: "0.40.0"
+version: 4.21.2
+appVersion: "0.39.2"
```

```suggestion
appVersion: "0.40.0"
```
The latest version is 0.40.0.